.eslintrc.json

```diff
@@ -22,6 +23,9 @@
         "variables": true
       }
     ],
+    "no-void": "off",
+    "no-param-reassign": "off",
```

Comment:
please can we allow this rule as i find it best to reduce side effects, especially in such a big codebase like this. can you point out where you had the problem with it? if no other solution we can just disable it for that line/lines